doc/reference/storages.rst

```diff
@@ -144,31 +144,46 @@ Options:
 Noteworthy non-included storages
 ================================
 
-A number of important ZODB storages are distributed separately, including:
+A number of important ZODB storages are distributed separately.
+
+Base storages
+-------------
+
+Unlike the included storages, all the implementations listed in this section
+allow multiple processes to share the same database.
+
+NEO
+  `NEO <https://lab.nexedi.com/nexedi/neoppod>`_ can spread data among several
+  computers for load-balancing and multi-master replication. It also supports
+  asynchronous replication to off-site NEO databases for further disaster
+  resistance without affecting local operation latency.
```

Comment:
I wasn't trying to criticize that as a drawback, just highlight that it's a substantial difference between that storage and the others. But then again, so is blob support.

You are right, it is quite clear from the linked page. So I withdraw my suggestion (unless someone else still feels like it needs to be included).